Initium-Odp/src/com/universeprojects/miniup/server/commands/CommandTransmuteItems.java

```diff
@@ -85,6 +85,8 @@ else if (recipes.size() == 1) {
 			List<Key> results = (List<Key>) recipe.getProperty("results");
 			CachedEntity resultItem = null;
 			
+			ds.beginTransaction();
```

Comment:
Have a look at the Transaction class. it lives in the CachedDatastore project. Then search for other usages of it to see how it works.